### PR TITLE
set main file structure

### PR DESCRIPTION
**Pull request by Bilkash**

### CODE REVIEW

File: src/pages/AuthPage/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './ui';
```
Comment (mmospanenko) [inline src/pages/AuthPage/index.ts]: change dir name

File: src/entities/session/model/events/index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/**
```
Comment (mmospanenko) [inline src/entities/session/model/events/index.tsx:1]: you can use store for this

File: src/entities/session/types.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export type UserType = {
```
Comment (mmospanenko) [inline src/entities/session/types.ts]: just User or CurrentUser

File: src/entities/session/ui/UserComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { UserType } from '../types';
+import { Container } from '../../../shared/ui';
+
+export const UserComponent = ({ id, name }: UserType) => {
```
Comment (mmospanenko) [inline src/entities/session/ui/UserComponent.tsx]: use object as namespace

File: src/feature/session/index.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export * from './logIn';
```
Comment (mmospanenko) [inline src/feature/session/index.ts]: wrong file naming

File: src/feature/session/logIn/model/index.tsx [hunk 6]
```diff
@@ -0,0 +1,3 @@
+/**
+ * handle for LoginButton here
```
Comment (mmospanenko) [inline src/feature/session/logIn/model/index.tsx]: index only for exports

File: src/feature/session/logIn/ui/LoginButton.tsx [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Button } from '../../../../shared/ui';
```
Comment (mmospanenko) [inline src/feature/session/logIn/ui/LoginButton.tsx]: too deep. Try to use aliases or base and relative paths

File: src/feature/session/logIn/ui/index.ts [hunk 8]
```diff
@@ -0,0 +1 @@
+export * from './LoginButton';
```
Comment (mmospanenko) [inline src/feature/session/logIn/ui/index.ts]: naming

**Review by mmospanenko — APPROVED** [review]

**Review by bstarchuk — APPROVED** [review]
use kebab case for folder and file naming

**Review by vliashenko — APPROVED** [review]

**Pull request merged**